shared/core/types/proto/ibc_events.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package core;
+
+option go_package = "github.com/pokt-network/pocket/shared/core/types";
+
+message Attribute {
```

Comment:
Please add comments (doesn't have to be super detailed, but something) explaining how/where/why these are used.